Added R6Stats presence

Hibanka opened the pull request:
Presence for [R6Stats](https://r6stats.com/)

![1](https://user-images.githubusercontent.com/48926119/108611894-d4d26880-73eb-11eb-9ef3-10d7b725043b.png)
![2](https://user-images.githubusercontent.com/48926119/108611895-d6039580-73eb-11eb-97ee-1895c8a8f659.png)
![3](https://user-images.githubusercontent.com/48926119/108611896-d69c2c00-73eb-11eb-9256-f4a70e660c13.png)
![4](https://user-images.githubusercontent.com/48926119/108611897-d69c2c00-73eb-11eb-8706-4501772a7529.png)

Bas950 commented on websites/R/R6Stats/presence.ts:
```diff
@@ -0,0 +1,35 @@
+    startTimestamp: browsingStamp
+  };
+
+  if (window.location.pathname.includes('/leaderboards')) {
+    presenceData.details = "Leaderboards";
```
shouldnt you do like viewing the leaderboards etc.?

Bas950 reviewed (changes requested):
Kinda low quality presence (not much to it)

Hibanka commented:
I rewrote some titles (Viewing...) and added more information for the leaderboards page (check screenshots below).
Also, some new buttons.

I think it's better now. Presence covers the entire site (except for the "premium" page which is not needed there).

There is nothing more to add.

![3](https://user-images.githubusercontent.com/48926119/108630379-3be43180-746d-11eb-8da6-3f6be7c77144.png)
![4](https://user-images.githubusercontent.com/48926119/108630382-3c7cc800-746d-11eb-8d80-1239f9af95e9.png)
![1](https://user-images.githubusercontent.com/48926119/108630385-3f77b880-746d-11eb-82ad-9bbd21959fc1.png)
![2](https://user-images.githubusercontent.com/48926119/108630386-3f77b880-746d-11eb-9e88-8103ccd563ff.png)
![5](https://user-images.githubusercontent.com/48926119/108630387-40104f00-746d-11eb-9b1b-6f33d5a73703.png)
